airflow/www/views.py

```diff
@@ -1741,16 +1742,44 @@ def _mark_task_instance_state(  # pylint: disable=too-many-arguments
         from airflow.api.common.experimental.mark_tasks import set_state
 
         if confirmed:
-            altered = set_state(
-                tasks=[task],
-                execution_date=execution_date,
-                upstream=upstream,
-                downstream=downstream,
-                future=future,
-                past=past,
-                state=state,
-                commit=True,
-            )
+            with create_session() as session:
+                altered = set_state(
+                    tasks=[task],
+                    execution_date=execution_date,
+                    upstream=upstream,
+                    downstream=downstream,
+                    future=future,
+                    past=past,
+                    state=state,
+                    commit=True,
+                    session=session,
+                )
+
+                if state == State.SUCCESS and resume:
+                    # If Resume is checked when marking success, clear downstream tasks that
+                    # are in upstream_failed state to resume them.
+
+                    # Flush the session so that the tasks marked success are reflected in the db.
+                    session.flush()
+                    subdag = dag.sub_dag(
```

Comment:
This is the deprecated name for it

```suggestion
                    partial_dag = dag.partial_subset(
```

Reply:
Thanks. Done